Packages/ChatSecure/ChatSecure/Classes/Signal/InMemoryStore/IdentityKeyInMemoryStore.swift

```diff
@@ -31,10 +31,13 @@ final class IdentityKeyInMemoryStore {
 			let currrentServer = channelStorage.realmManager.getCurrentServer()
 			let clientId = currrentServer?.profile?.userId ?? ""
 			let domain = currrentServer?.serverDomain ?? ""
+			if let tempKey = identityKeys[clientId+domain] {
+				return tempKey
+			}
 			let jsonDecoder = JSONDecoder()
 			if let keyData: Data = storage.object(forKey: clientId + domain, collection: .domain(domain)) {
 				let key = try jsonDecoder.decode(SignalIdentityKey.self, from: keyData)
-				identityKey = key
+				identityKeys[clientId+domain] = key
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/0b09249675dff5b0a460b69f27e9f3e1d518728a/Packages/ChatSecure/ChatSecure/Classes/Signal/InMemoryStore/IdentityKeyInMemoryStore.swift#L40"></span>Operators should be surrounded by a single whitespace when they are being used.<br />
<code>operator_usage_whitespace</code> <code>IdentityKeyInMemoryStore.swift:40</code></td>
    </tr>
  </tbody>
</table>